packages/muelu/src/Graph/Containers/MueLu_Aggregates_kokkos_def.hpp

```diff
@@ -180,6 +180,58 @@ namespace MueLu {
 
     return graph_;
   }
+  
+  template <class LocalOrdinal, class GlobalOrdinal, class DeviceType>
+  void 
+  Aggregates_kokkos<LocalOrdinal, GlobalOrdinal, Kokkos::Compat::KokkosDeviceWrapperNode<DeviceType> >::ComputeNodesInAggregate(LO_view & aggPtr, LO_view & aggNodes, LO_view & unaggregated) const {
+    LO numAggs  = GetNumAggregates();
+    LO numNodes = vertex2AggId_->getLocalLength();
+    auto vertex2AggId = vertex2AggId_->getDeviceLocalView(Xpetra::Access::ReadOnly);
+    typename aggregates_sizes_type::const_type aggSizes = ComputeAggregateSizes(true);
+    LO INVALID = Teuchos::OrdinalTraits<LO>::invalid();
+
+    aggPtr = LO_view("aggPtr",numAggs+1);
+    aggNodes = LO_view("aggNodes",numNodes);
```

Comment:
Is `numNodes` the number of all aggregated nodes, or the number of all nodes?

Reply:
The number of all nodes. This is the same way it's handled in `MueLu_Aggregates` from what I recall, meaning there's probably a small amount of padding at the end in the case where there are unaggregated nodes.